tests/foreman/cli/test_activationkey.py

```diff
@@ -49,1578 +49,1577 @@
 from robottelo.constants import REPOSET
 from robottelo.constants.repos import FAKE_0_YUM_REPO
 from robottelo.datafactory import invalid_values_list
+from robottelo.datafactory import parametrized
 from robottelo.datafactory import valid_data_list
 from robottelo.decorators import skip_if_not_set
 from robottelo.ssh import upload_file
-from robottelo.test import CLITestCase
 from robottelo.utils.issue_handlers import is_open
 from robottelo.vm import VirtualMachine
 
 
-class ActivationKeyTestCase(CLITestCase):
-    """Activation Key CLI tests"""
-
-    @classmethod
-    def setUpClass(cls):
-        """Tests for activation keys via Hammer CLI"""
-        super().setUpClass()
-        # syspurpose test will use cls org and manifest
-        cls.org = make_org(cached=True)
-        with manifests.clone() as manifest:
-            upload_file(manifest.content, manifest.filename)
-        try:
-            Subscription.upload({'file': manifest.filename, 'organization-id': cls.org['id']})
-        except CLIReturnCodeError as err:
-            raise CLIFactoryError(f'Failed to upload manifest\n{err.msg}')
-
-    @staticmethod
-    def get_default_env():
-        """Get default lifecycle environment"""
-        return LifecycleEnvironment.info(
-            {'organization-id': ActivationKeyTestCase.org['id'], 'name': 'Library'}
-        )
+@pytest.fixture(scope='module')
+def get_default_env(module_org):
+    """Get default lifecycle environment"""
+    return LifecycleEnvironment.info({'organization-id': module_org.id, 'name': 'Library'})
 
-    def _make_activation_key(self, options=None):
-        """Make a new activation key and assert its success"""
-        if options is None:
-            options = {}
 
-        # Use default organization if None are provided
-        no_org_flag = (
-            not options.get('organization', None)
-            and not options.get('organization-label', None)
-            and not options.get('organization-id', None)
-        )
-        if no_org_flag:
-            options['organization-id'] = self.org['id']
+@pytest.mark.tier1
+@pytest.mark.parametrize('name', **parametrized(valid_data_list()))
```

Comment:
I found a lot of these scattered in this file.  Usually when it's in regards to names, the use of `valid_names_list` is more preferred, but you can engage with someone on when to use `valid_data_list` vs `valid_name_list`.  Here's the list of lines that I found it in case you want to change them.

155
300
395
638
924
1030
1227

Reply:
The conversions are one to one with the original tests file. We can always go back and change the tests later if we decide they need to change